### DISCUSSION TITLE
Target .NET 6

### DISCUSSION THREAD

**Discussion by domingoladron** [gaelj/BlazorCodeMirror6#72] [Ideas]
Hey there,

Love the component.  However, with .NET 6 still in support (in fact, in support longer than .NET 7), I've locally tested a super minor PR to build a .NET 6 version of the package.

If you're keen, and I have a PR, would you be keen?  I can create a fork and you can pull that in if that's what you prefer. 

If you wish to test it yourself, the change is so small it's ridiculous.  Not sure if you need to change the gh action for building a .NET 6 package.

Let me know.

**Comment by pm64** [comment]
@domingoladron, the support you're referring to is a Microsoft thing, not a BlazorCodeMirror6 thing.

@gaelj may have good reasons for the current framework targets.

I think the etiquette would be to ask if there's a reason .NET 6 isn't targeted and whether the maintainer would be open to adding this target. Not "where do you want the PR?", come on man.

**Reply by domingoladron** [reply]
Then I rephrase my question.

Are you open to .NET 6 targeting?  And if so, I'm happy to provide a PR of said change.

**Comment by gaelj** [comment]
Hi ! I didn't add Net 6 simply because I personally don't use it anymore and I felt that now that 8 LTS is out, I could drop 6. But you are right, 6 will still be around officially for some time. You can submit your PR. Don't forget to update the README and add a conditional reference to Microsoft.AspNetCore.Components.Web version 6.

**Comment by domingoladron** [comment]
@gaelj ,

Cheers for your reply.  I've sent though the PR.  As I said, I've tested the change in a .NET 6 application and it does work as expected.  

https://github.com/gaelj/BlazorCodeMirror6/pull/73/files

Cheers and let me know if you want/need anything more from me.
